spec/UI/UI_spec.rb

```diff
@@ -34,7 +40,7 @@ def mark_board(marks, symbol)
 
   it 'announces a tie' do
     ui = UI.new(output, StringIO.new)
-    players = [Player.new("X", ui), Player.new("O", ui)]
+    players = [Human.new("X", ui), Easy_computer.new("O")]
```

Comment:
Variables should have underscores, but class names don't This would be something like `EasyComputer`.